packages/serverless-aws-cognito-user-pool/serverless.js

```diff
@@ -17,16 +17,24 @@ class ServerlessAwsCognito extends Component {
             this.context.instance.debug("Input was not changed, no action required.");
             return this.state.output;
         }
-
+        let schemaConfig = []
         const {
             region = "us-east-1",
             name,
             tags = {},
             appClients = [],
-            allowSignup = false
+            allowSignup = false,
+            Schema
```

Comment:
@Arun4you please make input keys for `UsernameAttributes` and `Schema` lower case, to be consistent with the existing components. `usernameAttributes` and `schema`.

Reply:
@Pavel910 Thank you for the feedback. I have committed the changes again. Please validate.